container-search/src/main/java/com/yahoo/prelude/query/Item.java

```diff
@@ -60,7 +60,8 @@
         GEO_LOCATION_TERM(27),
         TRUE(28),
         FALSE(29),
-        MULTI_TERM(30);
+        MULTI_TERM(30),
+        FUZZY(31);
```

Comment:
Sorry about the introduction of MULTI_TERM at the same time. To avoid conflicts we moved MULTI_TERM to use value 7, but that change came a bit late (https://github.com/vespa-engine/vespa/pull/21771) due to merge stop on factory. Please merge with master and change back to using value 30 for FUZZY. Only 5 bits are currently used to track item type, so we must use the remaining values wisely.